extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -408,7 +408,7 @@ class AmpLightbox extends AMP.BaseElement {
     this.triggerEvent_(LightboxEvents.OPEN, trust);
 
     this.getHistory_()
-      .push(this.close.bind(this))
+      .push((unused) => this.close(ActionTrust.HIGH))
```

Comment:
My hunch is this should not override the trust provided to the method.

`.push((unused) => this.close(trust))`

Adding @caroqliu for a second opinion.

Reply:
That's a good point. I was on the fence here, since this will be executed as a result of an explicit user action (clicking browser back). But I can see how it would also make sense to respect the provided trust.

I'll wait for @caroqliu's opinion before changing it.

Reply:
The method definition for `close` doesn't appear to gate based on a certain minimum trust, so it should not make a difference what the value is. Though I agree there is sufficient user intention to imply `ActionTrust.HIGH`, it may be more confusing to override the given trust without any change in end user experience.

Reply:
That sounds good. I've updated the code to pass in the provided trust value.